backend/services/auth.js

```diff
@@ -1,6 +1,26 @@
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import 'dotenv/config';
+import usersService from '../services/users.js';
+
 export default class AuthService {
-  static login() {
-    return 'You have successfully logged in!';
+  static login(email, password) {
+    const user = usersService.getUserByEmail(email);
+
+    if (user && bcrypt.compareSync(password, user.password)) {
+      const payload = {
```

Comment:
why do you need payload object, it's the same as user object ?

Reply:
I use payload object to create JWT and this token then has it as it's payload data. User object additionaly contains password field which I don't want JWT to contain. Also if in the future I would expand user entity with some additional fields they wouldn't be passed to JWT automatically, which would happen if I wouldn't have payload object and just use user object to create JWT. 

![image](https://user-images.githubusercontent.com/55427679/158554948-28279969-4545-4f3d-b5f1-41022bf5f155.png)